src/sagemaker/transformer.py

```diff
@@ -454,6 +454,7 @@ def transform_with_monitoring(
             monitor_before_transform=monitor_before_transform,
             supplied_baseline_constraints=supplied_baseline_constraints,
             supplied_baseline_statistics=supplied_baseline_statistics,
+            fail_on_violation=fail_on_violation,
```

Comment:
Is this change in default behavior going to impact any customer?

Reply:
current default is true 
we have added a variable so that customer can make it false if necessary

the default for the variable is `true` too